libs/math/transform.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2015, Simon Fuhrmann, Nils Moehrle
+ * TU Darmstadt - Graphics, Capture and Massively Parallel Computing
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD 3-Clause license. See the LICENSE.txt file for details.
+ */
+
+#ifndef MATH_TRANSFORM_HEADER
+#define MATH_TRANSFORM_HEADER
+
+#include "math/matrix.h"
+#include "math/matrix_svd.h"
+
+MATH_NAMESPACE_BEGIN
+
+template <typename T, int N>
```

Comment:
Can you please add a function declaration with a comment what this function is doing?
An then separate the declaration from the definition with `/* --- Implementation --- */` like in the other header files.